src/data/relicansData.js

```diff
@@ -0,0 +1,379 @@
+import React from 'react';
```

Comment:
It doesn't look like you're rendering any react components here. Do you need this import?

Reply:
It required it because of the <p> tags in the bios!

Reply:
I'm so dumb 😅 . How did I not see that?!?!!